### PR TITLE
Statements shortcode

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Added a shortcode for displaying ucf_statements from a remote REST API endpoint.

**Motivation and Context**
We want to be able to display the statements listed on news in various other places.

**How Has This Been Tested?**
The shortcode is being used on the newsroom page in DEV on the president's site.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: layouts/ucf-statements-classic.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		<article class="mb-4">
+			<a href="<?php echo $item->link; ?>" target="_blank" rel="nofollow">
+				<h3 class="h6 text-secondary ucf-statement-title"><?php echo $item->title->rendered; ?></h3>
+			</a>
+			<?php if ( $item->tu_author ) : ?>
+			<cite class="ucf-statement-author text-muted text-small">
+				<?php echo $item->tu_author->name; ?>
+			</cite>
+			<?php endif; ?>
+			<div class="ucf-statement-date">
```
Comment (cjg89) [inline layouts/ucf-statements-classic.php:48]: `<time>` could be nice to use here instead of `<div>` for added semantic value

**Review by cjg89 — APPROVED** [review]

**Review by rjucf — APPROVED** [review]

File: layouts/ucf-statements-classic.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	add_filter( 'ucf_statements_classic_before', 'ucf_statements_classic_before', 10, 3 );
+}
+
+if ( ! function_exists( 'ucf_statements_classic_title' ) ) {
+	function ucf_statements_classic_title( $content, $items, $args ) {
+		ob_start();
+
+		if ( isset( $args['title'] ) && ! empty( $args['title'] ) ) :
+	?>
+		<h2 class="h5 text-uppercase text-default-aw mb-4"><?php echo $args['title']; ?></h2>
```
[the remark this answers is not in the document]
Comment (jmbarne3) [reply layouts/ucf-statements-classic.php:23]: Updated the Wiki.

File: layouts/ucf-statements-classic.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+if ( ! function_exists( 'ucf_statements_classic_content' ) ) {
+	function ucf_statements_classic_content( $content, $items, $args ) {
+		ob_start();
+
+		foreach( $items as $item ) :
+	?>
+		<article class="mb-4">
+			<a href="<?php echo $item->link; ?>" target="_blank" rel="nofollow">
+				<h3 class="h6 text-secondary ucf-statement-title"><?php echo $item->title->rendered; ?></h3>
```
[the remark this answers is not in the document]
Comment (jmbarne3) [reply layouts/ucf-statements-classic.php]: Updated to use a `strong`, but added `d-block` so it behaves like a block element.

File: layouts/ucf-statements-classic.php (hunk 1 above)
Comment (jmbarne3) [reply layouts/ucf-statements-classic.php]: Updated to use `time` element and added the `datetime` attribute.

**Review by cadie — APPROVED** [review]

**Pull request merged**